form-page.js

```diff
@@ -0,0 +1,98 @@
+//getting element(s) from html
+const form = document.querySelector('[data-js="form"]');
+
+//create event listener
+form.addEventListener("submit", (event) => {
+  //prevent default behavior of event listener
+  event.preventDefault();
+
+  // Read all entered data from the input fields
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+
+  // Generate new DOM element for a card with createElement()
+  const newCard = document.createElement("ul");
+
+  //Insert the form's data as text into the DOM elements
+  newCard.innerHTML = `<li class="card-list__item">
+          <article class="card">
+            <h2 class="card__question">
+              ${data.newQuestion}
+            </h2>
+            <button
+              class="card__button-answer"
+              data-js="answer-button"
+              type="button"
+            >
+              Show answer
+            </button>
+            <p class="card__answer card__answer--active" data-js="answer">${data.newAnswer}</p>
+            <ul class="card__tag-list">
+              <li class="card__tag-list-item">#${data.tagField}</li>
+            </ul>
+            <div class="card__button-bookmark">
+              <button
+                class="bookmark"
+                data-js="bookmark-button"
+                aria-label="bookmark"
+                type="button"
+              >
+                <svg
+                  class="bookmark__icon"
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewbox="0 0 24 24"
+                >
+                  <path
+                    d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </article>
+        </li>`;
+
+  //Add style to newly created card
+  newCard.classList.add("card-list");
+
+  //Append the card to the page, directly below the form
+  form.append(newCard);
```

Comment:
I think, if you use 
document.body
 it will be appended in same plane as the form Element - directly below the form. But if you use 
form.append
it will be appended as a child of the form Element.

```suggestion
  document.body.append(newCard);
```

Reply:
Hey there;

Thank you for your feedback! :-)

I tried to append the new stuff both to the form and to the document. Both versions didn't look good. I think "main" is the right place to append. :-)

C u on Monday!